napari/layers/points/points.py

```diff
@@ -149,6 +155,12 @@ class Points(Layer):
     size : array (N, D)
         Array of sizes for each point in each dimension. Must have the same
         shape as the layer `data`.
+    fixed_size : bool
+        If active, point sizes do not change when zooming.
+    antialias : float
+        If non-zero, defines the width in pixels of antialiasing.
+    spherical : bool
+        Render lighting and shading on points to give a 3D spherical appearance.
```

Comment:
I wonder if we need to expose all of these on the points layer? I guess we might, but I do worry about bloat of the class a little. Curious if there are any alternatives people could imaging here? cc @jni